pop-api/examples/fungibles/tests.rs

```diff
@@ -12,21 +14,44 @@ use pop_api::{
 	},
 	v0::fungibles::events::{Approval, Created, Transfer},
 };
-use pop_sandbox::{Balance, DevnetSandbox as Sandbox, ALICE, BOB};
 use scale::Encode;
-use std::sync::LazyLock;
 use utils::*;
 
-#[drink::contract_bundle_provider]
-enum BundleProvider {}
+use super::*;
 
+const UNIT: Balance = 10_000_000_000;
+const INIT_AMOUNT: Balance = 100_000_000 * UNIT;
+const INIT_VALUE: Balance = 100 * UNIT;
+const ALICE: AccountId32 = AccountId32::new([1u8; 32]);
+const BOB: AccountId32 = AccountId32::new([2_u8; 32]);
+const CHARLIE: AccountId32 = AccountId32::new([3_u8; 32]);
 const AMOUNT: Balance = MIN_BALANCE * 4;
 const MIN_BALANCE: Balance = 10_000;
 const TOKEN: TokenId = 1;
 
 static CONTRACT: LazyLock<ContractBundle> = LazyLock::new(|| BundleProvider::local().unwrap());
 
-/// Deployment and constructor method tests.
+/// Sandbox runtime environment for `devnet` runtime.
+pub struct Sandbox {
+	ext: TestExternalities,
+}
+
+impl Default for Sandbox {
+	fn default() -> Self {
+		let balances: Vec<(AccountId32, u128)> =
```

Comment:
Assume the ext field name cant be changed, so can you add a comment here about initialisation genesis state so its easy for devs to understand what is going on